src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,65 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding: 30px 10px;
+  flex-wrap: wrap;
+  width: getWidth($width-cell, $cell-amount-row, $base-gap);
+  gap: 1px;
+
+  &--position {
+    margin: 0 auto;
+  }
+
+  $start-day:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $day, $marg-ratio in $start-day {
+    &-start-day--#{$day} &__day:first-child {
```

Comment:
The selector should be '&--start-day-#{$day} &__day:first-child' to follow the BEM naming convention.